### PR TITLE
New events

### PR DESCRIPTION
**Pull request by teresa-m**
Added two now events
- HPC bioNT workshop (external)
- HTS workshop freiburg

### CODE REVIEW

File: events/2025-03-10-hts-workshop-freiburg.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+contributions:
+  organisers: [Sch-Da, teresa-m]
+  instructors: [pavanvidem, Nilchia, teresa-m, wm75, EngyNasr, paulzierep]
+  funding: [eurosciencegateway, deNBI, nfdi4plants]
+
+location:
+  geo:
+    lat: 47.993460
+    lon: 7.845110
```
Comment (hexylena) [inline events/2025-03-10-hts-workshop-freiburg.md:31]: (fyi you don't have to fill this in actually! the automation will take care of it for you.)

File: events/2025-03-10-hts-workshop-freiburg.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  name: University Freiburg
+  # address: Werthmannstrasse 4
+  # postcode: 79104
```
Comment (hexylena) [inline events/2025-03-10-hts-workshop-freiburg.md:34]: i would suggest to include these, or is there a reason they are commented out?

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 1 above)
Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md:31]: Can I just add the street?

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 2 above)
Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md:34]: I think when I tried last time I got an error while building the site locally. But I will check again. Thanks for the hints.

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 1 above)
Comment (hexylena) [reply events/2025-03-10-hts-workshop-freiburg.md:31]: yeah if you just add the street address that's fine

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 2 above)
Comment (hexylena) [reply events/2025-03-10-hts-workshop-freiburg.md:34]: please let me know what the error is if you encounter it

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 1 above)
Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md:31]: So I could add just geo with nothing else like this:
```
location
  geo:
  name: University Freiburg
  ```

Comment (hexylena) [reply events/2025-03-10-hts-workshop-freiburg.md:31]: you don't need to add the geo key at all

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 2 above)
Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md]: Sorry, only an indentation error from my side. I added the Street now. 
About the geo location, I am not sure how to do this.

Comment (hexylena) [reply events/2025-03-10-hts-workshop-freiburg.md]: >  About the geo location, I am not sure how to do this.

yes, don't add the geo. You don't have to add it / do anything with it at all, please ignore it completely :)

Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md]: Hi @hexylena, thanks for your help first. I did not remove the geo, but now I am losing the location in my local build, which would be nice to have. 
Screenshot from 2024-10-15 14-06-34

Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md]: Should this also automatically apply using the address? If not I would like to keep the locations

**Review by hexylena — APPROVED** [review]

File: events/2025-03-10-hts-workshop-freiburg.md (hunk 2 above)
Comment (teresa-m) [reply events/2025-03-10-hts-workshop-freiburg.md]: @hexylena: Thanks for the explanation. Now I understand better what to do. Next time I will add the address and level to your automatication to add the geo location.

**Pull request merged**